### PR TITLE
Develop to Master

### PR DESCRIPTION
**Pull request by sujata-m**
- Adde Sample code to utilize `python-ms-core` package
- Added Fast API Project
- Created 2 sample APIs
- Added sample code to call `python-ms-core` package with `publish` and `subscribe` methods
- Removed monitor job
 - Added process message callback for we can receive message in that function
 - Refactor code

### CODE REVIEW

File: src/validation.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                        self.send_status(valid=False, upload_message=upload_message,
+                                         validation_message=f'No regex found in file {file_name}')
+        else:
+            print('No Message')
+
+    def send_status(self, valid: bool, upload_message: GTFSFlexUpload, validation_message: str = ''):
+        upload_message.data.is_valid = valid
+        upload_message.data.validation_message = validation_message
+        data = QueueMessage.data_from({
+            'messageId': '98383',
```
Comment (susrisha) [inline src/validation.py:39]: take out this magic number please.

File: src/serializer/gtfx_flex_serializer.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import json
+import datetime
+import decimal
+
+
+class Jsonable(object):
```
Comment (susrisha) [inline src/serializer/gtfx_flex_serializer.py:6]: Can we use modeller similar to the one in typescript with @Prop ??

**Review by susrisha — CHANGES_REQUESTED** [review]

File: src/validation.py (hunk 1 above)
Comment (sujata-m) [reply src/validation.py:39]: sure!

File: src/serializer/gtfx_flex_serializer.py (hunk 2 above)
Comment (sujata-m) [reply src/serializer/gtfx_flex_serializer.py:6]: I try to make it similar to @props

**Review by susrisha — APPROVED** [review]

**Pull request merged**